projects/demo/src/modules/components/input-card/index.html

```diff
@@ -43,5 +43,76 @@
         </tui-doc-example>
     </ng-template>
 
-    <tui-setup *pageTab="'Setup'" />
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <tui-textfield
+                [iconEnd]="icons.iconEnd"
+                [iconStart]="icons.iconStart"
+                [tuiTextfieldCleaner]="textfield.cleaner"
+                [tuiTextfieldSize]="textfield.size"
+            >
+                <label tuiLabel>Card number</label>
+                <input
+                    tuiInputCard
+                    tuiTextfield
+                    [autocomplete]="autocomplete"
+                    [disabled]="control.disabled"
+                    [icon]="icon"
+                    [invalid]="control.invalid"
+                    [readOnly]="control.readonly"
+                    [(ngModel)]="card"
+                    (binChange)="docAPIItemBinChange.emitEvent($event)"
+                />
+            </tui-textfield>
+        </tui-doc-demo>
+        <table tuiDocAPI>
+            <tr
+                name="[(ngModel)]"
+                tuiDocAPIItem
+                type="string"
+                [(value)]="card"
+            >
+                Card number (also works with a reactive control)
+            </tr>
+            <tr
+                name="[autocomplete]"
+                tuiDocAPIItem
+                type="boolean"
+                [(value)]="autocomplete"
+            >
+                Browser autocomplete of card
+            </tr>
+            <tr
+                name="[icon]"
+                tuiDocAPIItem
+                type="string | null"
+                [items]="iconVariants"
+                [(value)]="iconSelected"
+            >
+                Card icon
+            </tr>
+            <tr
+                #docAPIItemBinChange="docAPIItem"
```

Comment:
That's kind of long, maybe just `binChange`? :)

Reply:
Yeah, that's better